### PR TITLE
Bats tests fixes

### LINKED ISSUE #127: Bats tests need to always run in debug mode.
Regardless of whether `./configure` is called with `--enable-debug`, I think the bats tests should always run with full debug.

This requires updating `tests/functional/common.bash.in` to add `--debug` and a sensible `--global-log` value so aid debugging failures.

Bonus points for `cat`-ing the global log on test failure (think Travis).

### PR DESCRIPTION
**Pull request by jodh-intel**
Fixes #127.

### CODE REVIEW

File: tests/functional/help.bats [hunk 1]
```diff
@@ -21,127 +21,127 @@
 #Based on runc commands
 load common
 
-
 @test "cor -h" {
-  run $COR -h
+  eval run "$COR" -h
```
Comment (dlespiau) [inline tests/functional/help.bats]: hum could you explain in the commit message what is that needed? could we find a way to not have to eval every time?

**Comment by jodh-intel** [comment]
`eval`'s removed :)

**Comment by dlespiau** [comment]
yey! lgtm

**Comment by dlespiau** [comment]
ah no.

lgtm

**Comment by devimc** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/01org/cc-oci-runtime/pull-request/225/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=01org/cc-oci-runtime)

**Pull request merged**